chsdi/views/search.py

```diff
@@ -415,13 +415,20 @@ def _parse_locations(self, res):
             map(popAtrrs, attrs2Del)
         elif int(self.srid) not in (21781, 2056):
             self._box2d_transform(res)
-            try:
-                pnt = (res['y'], res['x'])
-                x, y = transform_coordinate(pnt, self.DEFAULT_SRID, self.srid)
-                res['x'] = x
-                res['y'] = y
-            except Exception:
-                raise exc.HTTPInternalServerError('Error while converting point(x, y) to EPSG:{}'.format(self.srid))
+            if int(self.srid) == 4326:
+                try:
+                    res['x'] = res['lon']
+                    res['y'] = res['lat']
+                except KeyError:
+                    raise exc.HTTPInternalServerError('Sphinx location has no lat/long defined')
```

Comment:
@ltclm , do you know if in sphinx response lat/lon are **always** defined?

Reply:
yes, they're always defined.